components/Layout/Header/Header.jsx

```diff
@@ -4,8 +4,9 @@ import Link from 'next/link';
 import { useRouter } from 'next/router';
 
 import { useContext } from 'react';
-import UserContext from 'components/UserContext/UserContext';
 import { getDataIncludes } from 'utils/saveData';
+
+import UserContext from 'components/UserContext/UserContext';
```

Comment:
Can we roll back these changes that are confusing? Same for the `_app` component